src/linux_mcp_server/tools/__init__.py

```diff
@@ -1 +1,44 @@
-"""Tool modules for Linux diagnostics."""
+from linux_mcp_server.tools.logs import get_audit_logs
```

Comment:
Call me picky, but I would prefer to have a new line separating the modules. It groups them visually so it's easier for humans to read. Not sure if that will interfere with linting/formatting though.

Reply:
The formatter will complain, but we can work around that. I believe in making formatters work _for_ us, not the other way around.